src/chrome/content/rules/DataFile.com-resources.xml

```diff
@@ -0,0 +1,23 @@
+<ruleset name="DataFile.com (resources)">
+
+	<target host="datafile.com" />
+	<target host="www.datafile.com" />
+	
+	<!-- Redirect to HTTP. Exclude implicit test URL -->
+	<exclusion pattern="^http://(www\.)?datafile\.com/$" />
+
+	<test url="http://datafile.com/fonts/helveticaneue-bold/HelveticaNeueCyrBold-webfont.eot" />
+	<test url="http://www.datafile.com/fonts/helveticaneue-bold/HelveticaNeueCyrBold-webfont.eot" />
+	<test url="http://www.datafile.com/images/backgrounds/bg_header.png" />
+	<test url="http://www.datafile.com/images/dflogo_en.png" />
+	<test url="http://www.datafile.com/images/icons/langs/en.png" />
+	<test url="http://www.datafile.com/css/images/ui-bg_highlight-soft_75_cccccc_1x100.png" />
+	<test url="http://www.datafile.com/css/getpremium_new.css" />
+	<test url="http://www.datafile.com/stylesheet/bundle_b68b43925235522902d187865fd2888a.css" />
+	<test url="http://www.datafile.com/javascript/bundle_ac996cee453148b2217dc9a861dbd8d3.js" />
+	<test url="http://www.datafile.com/js/windowAjax.js" />
+
+	<rule from="^http://(www\.)?datafile\.com/(fonts|images|css|javascript|js|stylesheet)/"
+		to="https://$1datafile.com/$2/" />
```

Comment:
`^` redirects to `www`, but HTTPS seems to work there. So we can simplify the rule by redirecting `^` to HTTPS and use complex regexes only for `www`.